io/net/network_test.py

```diff
@@ -274,6 +275,27 @@ def test_promisc(self):
             self.fail("failed to disable promisc mode")
         configure_network.ping_check(self.iface, self.peer, "5")
 
+    def test_ping6(self):
+        '''
+        IPV6 ping test to peer machine
+        '''
+        with ssh.Session(self.peer, user='root',
+                         password=self.peer_password) as session:
+            cmd = "ip addr show %s" % self.peer_interface
+            result = session.cmd(cmd)
+            if result.exit_status == 0:
+                result_ip = result.stdout_text
+                try:
+                    for line in result_ip.splitlines():
+                        if "inet6" in line:
+                            ipv6 = line.split()[1].split('/')[0]
+                except UnboundLocalError:
+                    if ipv6 == "":
+                        self.error("unable to get IPV6 of peer")
+                        return False
```

Comment:
@vaishnavibhat Would be better if you write a function for finding peer ipv6, similar to `get_peer_interface()` in configure_network,py, and then use that to get peer ipv6 here.

Reply:
> @vaishnavibhat Would be better if you write a function for finding peer ipv6, similar to `get_peer_interface()` in configure_network,py, and then use that to get peer ipv6 here.

Hi Narasimhan, 

I have opened a PR for the same but got some comments asking to use existing functions. Moreover I there is a comment saying configure_network() will get deprecated. So I have use made changes to network_test.py to test ipv6.

Reply:
Sure then